### ISSUE TITLE
All deDRMs including recent 10.0.3 not removing DRM and uploading as only acsm files in Calibre

### ISSUE THREAD

**Issue by MicG23** [noDRM/DeDRM_tools#248]
I was using Calibre 5.17 when my deDRM suddenly stopped working and all files were loading into Calibre as acsm, not epub.  I updated Calibre to 6.11 and installed the new deDRM 10.0.3 hoping that would fix the issue.  It didn't.  All files are still acsm only and "cannot be converted because no supported source format was found". 
I restarted in debug mode and this is the log I received when adding a book to Calibre.  Any suggestions on what the issue is?

ApplicationPaletteChange event ignored
Using calibre Qt style: True
ApplicationPaletteChange event received
calibre Debug log
calibre 6.11  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (10, 0, 3)
calibre 6.11  embedded-python: True
Windows-10-10.0.19045-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19045')
Python 3.10.1
Windows: ('10', '10.0.19045', 'SP0', 'Multiprocessor Free')
Interface language: None
Successfully initialized third party plugins: DeDRM (10, 0, 3)
QPA platform: windows
devicePixelRatio: 1.0
logicalDpi: 96.0 x 96.0
physicalDpi: 92.53889943074003 x 92.67567567567568
[0.00] Starting up...
[0.00] Showing splash screen...
[0.19] splash screen shown
[0.19] Initializing db...
[0.84] db initialized
[0.84] Constructing main UI...
Screens currently in system:
{'depth': 32,
 'device_pixel_ratio': 1.0,
 'geometry_in_logical_pixels': {'height': 1080, 'width': 1920, 'x': 0, 'y': 0},
 'index_in_screens_list': 0,
 'manufacturer': '',
 'model': '',
 'name': '\\\\.\\DISPLAY1',
 'serial': '',
 'size_in_logical_pixels': {'height': 1080, 'width': 1920},
 'virtual_geometry': {'height': 1080, 'width': 3840, 'x': -1920, 'y': 0}}
{'depth': 32,
 'device_pixel_ratio': 1.0,
 'geometry_in_logical_pixels': {'height': 1080,
                                'width': 1920,
                                'x': -1920,
                                'y': 0},
 'index_in_screens_list': 1,
 'manufacturer': '',
 'model': '',
 'name': '\\\\.\\DISPLAY2',
 'serial': '',
 'size_in_logical_pixels': {'height': 1080, 'width': 1920},
 'virtual_geometry': {'height': 1080, 'width': 3840, 'x': -1920, 'y': 0}}
Restoring geometry for: calibre_main_window_geometry
{'frame_geometry': {'height': 1048, 'width': 1920, 'x': 0, 'y': -8},
 'full_screened': False,
 'geometry': {'height': 1017, 'width': 1920, 'x': 0, 'y': 23},
 'maximized': True,
 'normal_geometry': {'height': 1017, 'width': 1920, 'x': 0, 'y': 23},
 'screen': {'depth': 32,
            'device_pixel_ratio': 1.0,
            'geometry_in_logical_pixels': {'height': 1080,
                                           'width': 1920,
                                           'x': 0,
                                           'y': 0},
            'index_in_screens_list': 0,
            'manufacturer': '',
            'model': '',
            'name': '\\\\.\\DISPLAY1',
            'serial': '',
            'size_in_logical_pixels': {'height': 1080, 'width': 1920},
            'virtual_geometry': {'height': 1080,
                                 'width': 3840,
                                 'x': -1920,
                                 'y': 0}}}
Matching screen: {'depth': 32,
 'device_pixel_ratio': 1.0,
 'geometry_in_logical_pixels': {'height': 1080, 'width': 1920, 'x': 0, 'y': 0},
 'index_in_screens_list': 0,
 'manufacturer': '',
 'model': '',
 'name': '\\\\.\\DISPLAY1',
 'serial': '',
 'size_in_logical_pixels': {'height': 1080, 'width': 1920},
 'virtual_geometry': {'height': 1080, 'width': 3840, 'x': -1920, 'y': 0}}
Setting widget geometry to: {'height': 1017, 'width': 1920, 'x': 0, 'y': 23}
Restoring widget to maximized
[2.00] main UI initialized...
[2.00] Hiding splash screen
Starting QuickView
[10.19] splash screen hidden
[10.19] Started up in 10.19 seconds with 13270 books
stdout+stderr from file dialog helper: [b'', b'']
piped data from file dialog helper: [b'\xcbO\x0e\xa79\xbb\x157\xf4\xbd7E\xfd\x85\n\x7f\x1e*Ay\x91\xec\x1d\xd4\x84\xa67\xf3\x0f X\xdc', b'C:\\Users\\Michelle\\Documents\\My Digital Editions.old\\TheCoupleatNumber9_9780063138162_7565654.acsm']
Added TheCoupleatNumber9 9780063138162 7565654 to db in: 0.1
Added 1 books in 0.6 seconds

**Comment by ElleKayEm** [comment]
You need to open the acsm file with ADE first.  Or you need the DeACSM plugin.

**Comment by MicG23** [comment]
Yes, I have done that.  I'm following the exact process I have done for years with no issues until now.

**Comment by ElleKayEm** [comment]
Import the epub into calibre, not the acsm file.

**Comment by MicG23** [comment]
Hi, I've always downloaded to ADE and then went to "Add books" in Calibre.  The file that I've always selected was an acsm and the deDRM plugin automatically took care of the transition to epub.  This is what is no longer working.  There is no "epub" file that I can locate, nor have I ever had to.  I'm asking for assistance for someone to read the log and provide further details.  I know the steps already, just trying to source what has caused the issue.  Thank you.

**Comment by ElleKayEm** [comment]
ADE is the program which handles downloading the epub or pdf.  After you've opened the acsm file, the epub or pdf is downloaded to your My Digital Editions folder.  That is what you add to calibre.  DeDRM then removes the DRM.  There is a plugin called DeACSM which allows you to skip using ADE altogether.  Then you can add the acsm file directly to calibre.  DeACSM will download the epub and then DeDRM will remove the DRM.

I see from the log that you add the acsm file and DeDRM is never triggered because it can do nothing with that on its own.

**Comment by MicG23** [comment]
But I want to use ADE editions because that's where all my downloads naturally go.  Where would I download them then?  I guess I'm trying to understand what has changed and why my previous method no longer works, rather than trying a completely new method.  I've always just used the deDRM, never had to use DeACSM

**Comment by MicG23** [comment]
I've located the folder with the epub files and can get them into Calibre and convert them.  However, there is no metadata being transferred anymore.  All fields are blank.  Are you able to see why this would be occurring?  Thanks!

**Comment by chocolatechipcats** [comment]
Did you use the DeACSM/ASCM Input plugin before? That would automatically create

> Hi, I've always downloaded to ADE and then went to "Add books" in Calibre. The file that I've always selected was an acsm and the deDRM plugin automatically took care of the transition to epub. This is what is no longer working. There is no "epub" file that I can locate, nor have I ever had to. I'm asking for assistance for someone to read the log and provide further details. I know the steps already, just trying to source what has caused the issue. Thank you.

Did you use the DeACSM/ASCM Input plugin before?

**Comment by ElleKayEm** [comment]
> I've located the folder with the epub files and can get them into Calibre and convert them. However, there is no metadata being transferred anymore. All fields are blank. Are you able to see why this would be occurring? Thanks!

Let's see a log of adding a book to calibre:  https://github.com/noDRM/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that

**Comment by junipermoss** [comment]
i'm also not able to remove DRM with either this plug in or the DeACSM plug in. when i import the file from the digital editions folder after opening the .ascm in ADE, the file is blank and/or still encrypted.

**Comment by ElleKayEm** [comment]
Post a log and we can take a look.

**Comment by junipermoss** [comment]
issue was resolved. turns out im a noob and didn't follow instructions properly. thanks though ! <3

**Comment by Patitapetit** [comment]
ADE only works with W8 or less. If you have Calibre 6, at least have W10

**Comment by ElleKayEm** [comment]
You mean calibre 6 doesn't work with Windows 8.  ADE works just fine with Windows 10.
